shop/src/Pages/HomePage/index.jsx

```diff
@@ -9,8 +9,11 @@ const HomePage = () => {
       <Banner />
       <div className="homePageContainer">
         <div className="cardsContainer">
-          <Card imgAndPathOfHomeCard={homePageCardContent} />
-          {/* Nothing change on it, that's why just display it and change routes. */}
+          {homePageCardContent.length > 0 ? (
+            <Card imgAndPathOfHomeCard={homePageCardContent} />
+          ) : (
+            <></>
```

Comment:
Better. But why don't we also show some sort of message to the user why there's no data as well
 
something like: "No Data Available" or "We can't what you're looking for".